app/helpers/pipeline_outputs_helper.rb

```diff
@@ -189,6 +189,26 @@ def get_s3_file(s3_path)
     end
   end
 
+  def get_presigned_s3_url(s3_path, filename)
+    s3 = Aws::S3::Resource.new(client: Client)
+    uri_parts = s3_path.split("/", 4)
```

Comment:
I saw the code that parses the URIs repeated in https://github.com/chanzuckerberg/idseq-web/blob/e70919eec874eecfdaa6d324959203279be7817d/app/controllers/samples_controller.rb#L1012 -- worth factoring it out into one of the helpers?

Reply:
Done! I found and replaced some other places where I saw this repeated.